test/slider-test.jsx

```diff
@@ -384,6 +384,22 @@ describe('Slider API', () => {
       assert(slider.getSnapPosition(96) === 100, 'position is at 100%');
       assert(slider.getSnapPosition(55) === 50, 'position is at 50%');
     });
+
+    /*
+      During slide, the animation requires getSnapPosition to not return
+      the position of the point, but rather the actual position of the slider.
+      This is accomplished with an extra arg to getSnapPosition.
+    */
```

Comment:
oh. this should go in the component not just in the test... it needs more explanation there, I think.